pkg/apis/pingcap/v1alpha1/tidbclusterautoscaler_types.go

```diff
@@ -85,6 +85,7 @@ type TidbClusterAutoScalerSpec struct {
 type TikvAutoScalerSpec struct {
 	BasicAutoScalerSpec `json:",inline"`
 
+	// Deprecated
```

Comment:
in which version you plan to remove these fields? you can add a note about this, e.g. `Deprecated in v1.1.3, planned for removal in v1.x.`

Reply:
updated